### DISCUSSION TITLE
Questions about using Trilium on true offline mode

### DISCUSSION THREAD

**Discussion by drawtheliner** [TriliumNext/Trilium#6231] [Q&A]
One of the main reasons that led me to Trilium is the possibility of having all my notes/data locally only  (ie. true offline). I've been testing the app and am still a bit confused about how this happens. Sorry if these questions are too basic - I'm not very tech savvy.

1. First of all, I'd like to be sure that I'm using it correctly. I downloaded the Windows package, unzipped it to a folder, and have been clicking on trilium.exe every time I want to run it (in other words: I never really installed it). Is this correct usage?

2. In the app, under Options/Sync, I'm told that if I leave the fields empty, "system proxy will be used". What does this mean exactly? Does it simply mean the app will be saving my data locally? If not, how do I go about achieving that? (This is mentioned in the documentation, under Synchronization, however it's too technical for me to fully understand.)

3. I understand Trilium keeps all my notes/data in a database (as opposed to text files). Where is this kept in my computer? Is it within the folder where I run the app from? I want to make sure I keep my data as I upgrade to new versions of the app..

Thanks!

**Comment by zadam** [comment]
Hello,

1. yes, this is correct. "Installation procedure" is to unzip the archive and run trilium.exe

2. The app is always storing the whole database locally. In Options/Sync, you can additionally configure that the app should sync changes to/from another instance. But even in such case, all instances always have complete copy of all data locally and thus available offline.

3. Location of the data depends on the OS, you can read a bit about this [here](https://github.com/zadam/trilium/wiki/Data-directory).

**Comment by drawtheliner** [comment]
Thank you for that!

I'd like to change the directory where my data is stored, to a folder less "buried".

Ï saw the "Changing the location of data directory" section, but couldn't really figure out how to accomplish that... If it's too advanced I'd rather not do it (though would appreciate an in-app UI in the future).

Thanks!

**Comment by meichthys** [comment]
[Trilium has entered maintenance mode](https://github.com/zadam/trilium/issues/4620#issuecomment-2094495698). Future enhancements will be addressed in TrilumNext: https://github.com/TriliumNext/trilium/issues/4994
